cruise-control/src/test/java/com/linkedin/kafka/cruisecontrol/monitor/task/LoadMonitorTaskRunnerTest.java

```diff
@@ -78,7 +78,13 @@ public void setUp() {
                                                                               false);
     AdminZkClient adminZkClient = new AdminZkClient(kafkaZkClient);
     for (int i = 0; i < NUM_TOPICS; i++) {
-      adminZkClient.createTopic("topic-" + i, NUM_PARTITIONS, 1, new Properties(), RackAwareMode.Safe$.MODULE$);
+      adminZkClient.createTopic(
+              "topic-" + i,
+              NUM_PARTITIONS,
+              1,
+              new Properties(),
+              RackAwareMode.Safe$.MODULE$,
+              false);
```

Comment:
This looks like another API change -- would switching to `AdminClient#createTopics` avoid this inconsistency?

Reply:
This is a good point, I will try and capture this in the next `adminZkClient` removal PR.